serializers/paginator.js

```diff
@@ -0,0 +1,33 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const { paginatedLink } = appRoot.require('/serializers/uri-builder');
+
+/**
+ * @summary Paginate data rows
+ * @function
+ * @param {[Object]} rows Data rows
+ * @param {Object} page Pagination query parameter
+ * @returns {*} Paginated data rows and pagination links
+ */
+const paginate = (rows, page) => {
+  const { number, size } = page;
+  const pageNumber = number ? parseInt(number, 10) : 1;
+  const pageSize = size ? parseInt(size, 10) : 10;
```

Comment:
Just curious, is it standard for our apis to always have default values of 10 and 1 for pageSize and pageNumber respectively? If not, is it something we should make configurable?

Reply:
Good idea. In [location frontend API](https://github.com/osu-mist/locations-frontend-api/blob/develop/swagger.yaml#L114-L126), the page[size] has default value 10 but page[number] doesn't. I don't think we have any standard for pagination parameters yet, and I just followed how we did before and give page[number] a default value 1 since I guess that makes sense. I am open to any suggestion. Let's talk about this in the group meeting as well :)